docs/install.md

```diff
@@ -5,78 +5,99 @@ feature_maturity: stable
 enterprise: 'no'
 ---
 
-Spark is available in the Universe and can be installed by using either the web interface or the DC/OS CLI.
+Spark is available in the Universe and can be installed by using either the GUI or the DC/OS CLI.
 
-##  <a name="install-enterprise"></a>Prerequisites
```

Comment:
@mgummelt line 10 is being removed (pink highlighting). are we talking about the same line?